utils/debug/Debug.js

```diff
@@ -133,37 +145,101 @@ function Debug(view, viewerDiv) {
             }
         }
 
+        if (displayed) {
+            nbVisibleChart.update();
+        }
+    }
+}
+
+function ThreeStats(chartDiv, width, left) {
+    createChartContainer(chartDiv, width, left, 'three-info');
 
+    const timestamp = Date.now();
+    const textureDataset = { label: 'texture count', data: [{ x: 0, y: 0 }] };
+    const geometryDataset = { label: 'geometry count', data: [{ x: 0, y: 0 }], borderColor: 'rgba(75,192,192,1)' };
+    const label = ['0s'];
+    const chart = new Chart('three-info', {
+        type: 'line',
+        data: {
+            labels: label,
+            datasets: [textureDataset, geometryDataset],
+        },
+        options: {
+            animation: { duration: 10 },
+            scales: {
+                yAxes: [{
+                    display: true,
+                    ticks: {
+                        suggestedMin: 0, // minimum will be 0, unless there is a lower value.
+                    },
+                }],
+            },
+        },
+    });
+
+    this.update = (displayed, view) => {
         // update line graph
         // update time
         const limit = 60;
         const timeInS = Math.floor((Date.now() - timestamp) / 1000);
         const lbl = `${timeInS}s`;
-        const identical = (viewInfoChartLabel.lastValidCompareIndex > 0 && viewInfoChartLabel[viewInfoChartLabel.lastValidCompareIndex] == lbl);
+        const identical = (label.lastValidCompareIndex > 0 && label[label.lastValidCompareIndex] == lbl);
         if (identical) {
-            viewInfoChartLabel.push('');
+            label.push('');
         } else {
-            viewInfoChartLabel.push(lbl);
-            viewInfoChartLabel.lastValidCompareIndex = viewInfoChartLabel.length - 1;
+            label.push(lbl);
+            label.lastValidCompareIndex = label.length - 1;
         }
 
-        if (viewInfoChartLabel.length > limit) {
-            viewInfoChartLabel.shift();
-            viewInfoChartLabel.lastValidCompareIndex--;
+        if (label.length > limit) {
+            label.shift();
+            label.lastValidCompareIndex--;
         }
 
-        viewLevelStartDataset.data.push({ x: timeInS, y: updateStartLevel });
-        viewUpdateDurationDataset.data.push({ x: timeInS, y: updateDuration });
-        if (viewLevelStartDataset.data.length > limit) {
-            viewLevelStartDataset.data.shift();
-            viewUpdateDurationDataset.data.shift();
+        const memory = view.mainLoop.gfxEngine.renderer.info.memory;
+        textureDataset.data.push({ x: timeInS, y: memory.textures });
```

Comment:
`memory` is the name of the variables in `three.js` - I just used a local var to avoid writing: 
` view.mainLoop.gfxEngine.renderer.info.memory.textures` :)